core/node/config/config.go

```diff
@@ -81,13 +149,23 @@ type Config struct {
 	// Disable base chain contract usage.
 	DisableBaseChain bool
 
+	// Chains provides a map of chain IDs to their provider URLs as
+	// a comma-serparated list of chainID:URL pairs.
+	// It is parsed into ChainsString variable.
+	Chains string `dlog:"omit" json:"-"`
+
+	// This is comma-separated list chaidID:blockTimeDuration pairs.
+	// I.e. 0:12s,84532:2s,6524490:2s
+	ChainBlocktimes string
+
+	ChainConfigs map[uint64]*ChainConfig `mapstructure:"-"` // This is a derived field from Chains.
+
 	// xChain configuration
-	ChainsString                  string            `mapstructure:"chains"`
```

Comment:
Operators have written their build scripts to inject chainId, rpcUrls to ChainsString not Chains so removing this will break their next build. If you do not have a good reason to remove this config var named as such, please swap Chains for ChainsString to ensure we do not break operator builds and documentation.

Reply:
added ChainsString as fallback